assets/styles/app.css

```diff
@@ -127,3 +128,41 @@ footer a {
     color: gray;
     flex: 1;
 }
+
+article {
+    margin-left: 25%;
+    margin-right: 25%;
```

Comment:
Ça ressemble à une tentative de centrage, sauf que ça n'exprime pas du centrage : le code met des marges de 25% de la largeur du conteneur à gauche et à droite. Sur mobile, ça passe pas :/

Centrage qui marche sur mobile :

```
margin-left: auto;
margin-right: auto;
```

Ou :

```
margin: initial auto;
```

Pour que ça marche, le bloc doit arrêter de prendre toute la largeur de son parent à partir d'une certaine taille d'écran :

```
max-width: 65ch; /* environ 80 caractères avec les fonts les plus courantes */
```

https://www.smashingmagazine.com/2014/09/balancing-line-length-font-size-responsive-web-design/#line-length-measure-and-reading